components/inputs/docs/styling-native-inputs.md

```diff
@@ -0,0 +1,153 @@
+# Styling Native Inputs
```

Comment:
I'm de-emphasizing the styling of native inputs stuff by moving it into its own file. I don't think I'm going to surface this on the Daylight Site, since I think we want to encourage folks to use the web components now for the reasons outlined here. I'll move the other similar documentation (e.g. text inputs) in here as I get to them.